api_app/analyzers_manager/migrations/0094_analyzer_config_detectiteasy.py

```diff
@@ -0,0 +1,124 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": None,
+        "module": "detectiteasy.DetectItEasy",
+        "base_path": "api_app.analyzers_manager.file_analyzers",
+    },
+    "name": "DetectItEasy",
+    "description": "[DetectItEasy](https://github.com/horsicq/Detect-It-Easy) is a program for determining types of files.",
+    "disabled": False,
+    "soft_time_limit": 60,
+    "routing_key": "default",
+    "health_check_status": True,
+    "type": "file",
+    "docker_based": False,
+    "maximum_tlp": "CLEAR",
+    "observable_supported": [],
+    "supported_filetypes": [
```

Comment:
shouldn't this support all files?

Reply:
![image](https://github.com/intelowlproject/IntelOwl/assets/102874321/6048706f-0a65-4f0f-9b9d-de0c425dbc39)

I misunderstood this. You are right. I should make the list empty.